DEPENDENCIES.md

```diff
@@ -1,7 +1,11 @@
 | Name | Version | License | Used by | Env | Purpose |
 | --- | --- | --- | --- | --- | --- |
 | [black](https://pypi.org/project/black/) | 19.3b0 | MIT License (MIT) | Makefile | dev | Python code formater |
+| [bumpversion](https://pypi.org/project/bumpversion/) | 0.6.0 | MIT License (MIT license) | Makefile | dev | Python project version updater |
```

Comment:
The version doesn't match the one that we are actually using

Reply:
Correct, even the dependency is incorrect, was the first dependency used for this change, but then the bump2version was preferred.